alerts/resource_alerts.libsonnet

```diff
@@ -100,7 +100,7 @@
               kube_resourcequota{%(prefixedNamespaceSelector)s%(kubeStateMetricsSelector)s, type="used"}
                 / ignoring(instance, job, type)
               (kube_resourcequota{%(prefixedNamespaceSelector)s%(kubeStateMetricsSelector)s, type="hard"} > 0)
-                >= 1
+                = 1
```

Comment:
I don't think this i what you wanted.
The alert that you're trying to introduce is called "exceeded" so i assume it should be `>` instead of `=`